### PR TITLE
fixing report.Rmd

### PR DESCRIPTION
**Pull request by emchristensen**
initially, I had hard-coded the forecast file from the predictions folder.  Now we have the newmoons/year/month issue worked out, I changed report.Rmd so it finds the most recent forecast file made before the latest data in the repo, to make the plot

### CODE REVIEW

**Comment by emchristensen** [comment]
@gmyenni I can't figure out why the build is failing -- does it have to do with it can't find the htmltab package?  How do we make it find the package?  I added a library() call it's still failing.

**Comment by sdtaylor** [comment]
That is weird cause it totally ran before

File: models/model_functions.R [hunk 1]
```diff
@@ -2,6 +2,7 @@ library(lubridate)
 library(dplyr)
 library(magrittr)
 library(rmarkdown)
+library(htmltab)
 source('forecast_tools.R')
 
 get_moon_data <- function(){
```
Comment (sdtaylor) [inline models/model_functions.R:8]: try changing the `htmltab(....)` line to `htmltab::htmltab(....)`

**Comment by sdtaylor** [comment]
maybe it needs to be added into the  `install-packages.R` script?

**Pull request merged**